hack/build-and-push-validator.sh

```diff
@@ -0,0 +1,15 @@
+#!/bin/bash
+
+set -e
+
+VERSION="${1:-devel}"
+
+if [ -z "$QUAY_BOT_PASS" ] || [ -z "$QUAY_BOT_USER" ]; then
+	echo "missing QUAY_BOT_{USER,PASS} env vars"
+	exit 1
+fi
+
+echo "$QUAY_BOT_PASS" | docker login -u="$QUAY_BOT_USER" --password-stdin quay.io
+docker build -t quay.io/kubevirt/kubevirt-template-validator:$VERSION ./internal/template-validator/
+docker push quay.io/kubevirt/kubevirt-template-validator:$VERSION
+docker logout quay.io
```

Comment:
The logic in this file is not needed now that the validator is part of ssp-operator repository.

Can you create a new commit, that would move the `docker build` and `docker push` commands to new Makefile targets, and remove this file? It is no longer needed to login to quay, because our Makefile already expects that the user is logged in. The github release scripts login before running build.

It would also be useful to extract the path to an env varaible in the makefile. Similarly to how SSP operator is build.